src/index.ts

```diff
@@ -39,17 +67,94 @@ async function run(): Promise<void> {
 
     await Deploy.getDeployment(appSelector, namespaceArg);
 
-    let route = await Deploy.getRoute(appName, namespaceArg);
     // To make it appear as a URL
-    route = `http://${route}`;
+    const route = `http://${await Deploy.getRoute(appName, namespaceArg)}`;
     ghCore.info(`✅ ${appName} is exposed at ${route}`);
 
     ghCore.setOutput(Outputs.ROUTE, route);
     ghCore.setOutput(Outputs.SELECTOR, appSelector);
 }
 
+async function createPullSecretFromAuthFile(createPullSecretFrom: string): Promise<void> {
```

Comment:
there needs to be an else case, for if the user passes something other than `docker` or `podman`. if that is checked before this function call, then the type of this function's parameter should be a union `"docker" | "podman"`